clients/cloud/ccloud/ccloud-ccsr-example.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+source ../../../utils/helper.sh
+source ../../../utils/ccloud_library.sh 
+
+ccloud::validate_expect_installed \
+  && print_pass "expect installed" \
+  || exit 1
+check_timeout \
+  && print_pass "timeout installed" \
+  || exit 1
+ccloud::validate_version_ccloud_cli 1.13.0 \
+  && print_pass "ccloud version ok" \
+  || exit 1
+ccloud::validate_logged_in_ccloud_cli \
+  && print_pass "logged into ccloud CLI" \
+  || exit 1
+
+if [[ -z "$SR_API_KEY" ]] || [[ -z "$SR_API_SECRET" ]] ; then
+  echo "ERROR: You must export SR_API_KEY and SR_API_SECRET before running this script"
```

Comment:
nit ⚠️ : It's not required that they be "exported", but just that they be "set"
```suggestion
  echo "ERROR: The SR_API_KEY and SR_API_SECRET variables are required to be set when running this script"
```

Reply:
Addressed in separate commit